packages/react-scripts/template/README.md

```diff
@@ -301,6 +302,47 @@ becomes this:
 
 There is currently no support for preprocessors such as Less, or for sharing variables across CSS files.
 
+## Adding CSS Preprocessor (Sass, Less etc.)
+
+CSS preprocessors have become a vital part of build processes. Using a preprocesssor of your choice in a project bootstrapped using create-react-app, is fairly straightforward to setup, even without having to eject.
+
+First, install preprocessor of your choice. Sass seems the most popular weapon of choice at the moment, so we'll use it as an example.
```

Comment:
There's no need to point out sass' popularity.

`First, install the preprocessor of your choice. For example, with sass:`